napari/conftest.py

```diff
@@ -759,3 +773,47 @@ def pytest_runtest_setup(item):
                 "mock_console",
             ]
         )
+
+
+from datetime import timedelta  # noqa: E402
```

Comment:
Why not just import these at the top instead of ignoring E402? Or consider putting the terminal report in its own file/module if you think it's useful to keep them close.